sync/rest/document-permissions/delete-permission/delete-permission.3.x.js

```diff
@@ -5,7 +5,7 @@ var Twilio = require('twilio').Twilio;
 var client = new Twilio(accountSid, authToken);
 var service = client.sync.services('ISXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX');
 
-service.documents('MyFirstDocument')
+service.documents('ETXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX')
```

Comment:
@dr0nius Thanks for the update. FYI, we just updated the sample based on the _curl_ file, so if what you say is right, then it would be better to update the curl example too, to use `MyFirstDocument` instead (and all the examples to follow the same approach).

Wdyt?